src/api/python/hidra/utils/utils_netgroup.py

```diff
@@ -0,0 +1,31 @@
+
+from ctypes import *
+
+def find_hostname(netgroup):
+
+    libc = CDLL("libc.so.6")
+    if (libc != None):
+        pass
+    else:
+        libc = CDLL("libc.so.7")
```

Comment:
Is there a better way that going through a fixed list of library names? Why are `"libc.so.5"` or `libc.so.8` not included in that list?

Also it is probably better to not do the lookup at every function call but only once at module load time.